ouroboros-consensus-cardano-test/test/Test/ThreadNet/Cardano.hs

```diff
@@ -554,7 +555,10 @@ mkProtocolCardanoAndHardForkTxs
               TriggerHardForkAtVersion babbageMajorVersion
           }
         ProtocolTransitionParamsShelleyBased {
-            transitionTranslationContext = error "Conway.degenerateConwayGenesis"
```

Comment:
TODO This seems like it's in the wrong commit -- it should be in the first commit instead.

Reply:
Ugh, I squashed all the commits before rebasing on top of `master`, should I discard my local branch?

Reply:
Squashing probably addresses my concern here. 👍 resolved